packages/edit-widgets/src/components/save-button/index.js

```diff
@@ -26,9 +26,8 @@ function SaveButton() {
 		<Button
 			variant="primary"
 			isBusy={ isSaving }
-			aria-disabled={ isSaving }
+			aria-disabled={ isSaving || ! hasEditedWidgetAreaIds }
 			onClick={ isSaving ? undefined : saveEditedWidgetAreas }
```

Comment:
I think the condition here also needs to be updated. A user should not be able to click on the button when `! hasEditedWidgetAreaIds`.

PS. We should probably define `isDisabled` condition.

Reply:
Thanks for the review! I updated via 16282ad, does this make sense?